packages/react-native/Libraries/AppDelegate/RCTAppDelegate.mm

```diff
@@ -121,7 +134,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [self unstable_registerLegacyComponents];
     [RCTComponentViewFactory currentComponentViewFactory].thirdPartyFabricComponentsProvider = self;
 #endif
-    NSDictionary *initProps = [self prepareInitialProps];
+    NSDictionary *initProps = updateInitialProps([self prepareInitialProps], [self fabricEnabled]);
```

Comment:
CI tests failed because `fabricEnabled` only exists when `RCT_NEW_ARCH_ENABLED` is set :(

Feel free to also refactor this to create `initProps` before the `enableBridgeless` check.